### ISSUE TITLE
 irregular R-R interval

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9605]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
"Multifocal atrial tachycardia (MAT) is defined as irregular P-P, R-R, and P-R intervals,
isoelectric baseline between P waves, and ventricular rate over 100 beats/min"
PMID 35813391

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA 3282 Multifocal atrial tachycardia (MAT)

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001086
- label: Irregular RR interval
- parents: [Abnormal EKG (HP:0003115)](https://hpo.jax.org/app/browse/term/HP:0003115)
- pmid: PMID:35813391|PMID:30725696
- def: Abnormal increased variability in the timing between consecutive R-waves on an electrocardiogram.
- comment: Sinus arrhythmia is a variation of normal sinus rhythm that characteristically presents with an irregular rate in which the change in the R-R interval is greater than 0.12 seconds.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9605
- synonyms:
